airbyte-webapp/src/packages/cloud/services/workspaces/CloudWorkspacesService.tsx

```diff
@@ -108,22 +107,20 @@ export function useRemoveWorkspace() {
 export function useGetCloudWorkspace(workspaceId: string): CloudWorkspace {
   const service = useGetWorkspaceService();
 
-  return useSuspenseQuery<CloudWorkspace>([workspaceKeys.detail(workspaceId)], () => service.get(workspaceId));
+  return useSuspenseQuery<CloudWorkspace>(workspaceKeys.detail(workspaceId), () => service.get(workspaceId));
 }
 
 export function useInvalidateCloudWorkspace(workspaceId: string): () => Promise<void> {
   const queryClient = useQueryClient();
 
   return useCallback(
-    () => queryClient.invalidateQueries([workspaceKeys.detail(workspaceId)]),
+    () => queryClient.invalidateQueries(workspaceKeys.detail(workspaceId)),
     [queryClient, workspaceId]
   );
 }
 
-export function useGetUsage(workspaceId: string): CloudWorkspaceUsage {
+export function useGetCloudWorkspaceUsage(workspaceId: string): CloudWorkspaceUsage {
```

Comment:
Should we convert these to const exports? I feel like eslint should flag it.
While you were in here were you able to identify the difference between this code and our normal code? I saw them both but wasn't sure what the difference was.

Reply:
I'll update these manually. There is a lint rule to fix that needs to be introduced. It was excluded because the changes were not trivial in some areas. I'll see if they can be re-introduced.

Reply:
I'll look into the eslint rule and fix them separately.